client/post-editor/media-modal/detail/detail-preview-videopress.jsx

```diff
@@ -90,6 +128,34 @@ class EditorMediaModalDetailPreviewVideoPress extends Component {
 		}
 	}
 
+	play() {
+		if ( ! this.player || ! this.player.state ) {
+			return;
+		}
+
+		if ( typeof this.player.state.play === 'function' ) {
+			this.player.state.play();
+		}
+	}
+
+	pause() {
+		if ( ! this.player || ! this.player.state ) {
+			return;
+		}
+
+		if ( typeof this.player.state.pause === 'function' ) {
+			this.player.state.pause();
+		}
```

Comment:
This is also fixed in https://github.com/Automattic/wp-calypso/pull/11653/commits/a916c539900870d4e7b03c4bf0b09b00afaa05b8.